libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -481,6 +482,14 @@ private IdentityClaim getIdentityClaim(String credential)
         return new IdentityClaim(names, birthDates);
     }
 
+    public VectorOfTrust getVot(SignedJWT credential)
+            throws JsonProcessingException, IllegalArgumentException {
+        var votJson =
+                objectMapper.readTree(credential.getPayload().toString()).path(VOT_CLAIM_NAME);
```

Comment:
Do you think `VectorOfTrust.valueOf(credential.getJWTClaimsSet().getStringClaim(VOT_CLAIM_NAME));` is cleaner here - doesn't require us to do any explicit JSON mapping.